compiler/codegen/OMRCodeGenerator.cpp

```diff
@@ -3158,37 +3158,47 @@ void OMR::CodeGenerator::addRelocation(TR::Relocation *r)
 
 void OMR::CodeGenerator::addAOTRelocation(TR::Relocation *r, const char *generatingFileName, uintptr_t generatingLineNumber, TR::Node *node, TR::AOTRelocationPositionRequest where)
    {
-   TR_ASSERT(generatingFileName, "AOT relocation location has improper NULL filename specified");
+   self()->addExternalRelocation(r, generatingFileName, generatingLineNumber, node, static_cast<TR::ExternalRelocationPositionRequest>(where));
+   }
+
+void OMR::CodeGenerator::addAOTRelocation(TR::Relocation *r, TR::RelocationDebugInfo* info, TR::AOTRelocationPositionRequest where)
```

Comment:
are you keeping these around until downstream projects can be changed to use the new name?

Reply:
Ah! just saw the comment in the commit message. Other than the IP check, I have no concerns with this PR.